.github/workflows/Receive_build_and_test_on_push.yml

```
@@ -0,0 +1,27 @@
+    paths:
+      - 'Frends.MQTT.Receive/**'
+  workflow_dispatch:
+
+jobs:
+  build:
+    uses: FrendsPlatform/FrendsTasks/.github/workflows/linux_build_test.yml@main
+    with:
+      workdir: Frends.MQTT.Receive
+      dotnet_version: 8.0.x
+      prebuild_command: |
+    
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)